Rewrite send response BBEs (#3769)
ballerina-platform/ballerina-distribution: examples/http-send-response/http_send_response.md

# REST service - Send response

> Returning an `anydata` type from the resource method results in an HTTP response where the returned value will become the body. If the returned type is `nil`, then a `202 Accepted` response will be returned to the client without the body. Otherwise, the response will have the returned value as the body, and the `Content-type` header will be inferred from the return type. In addition, the response will be `201 Created` for `POST` resources and `200 Ok` for other resources. Furthermore, the `@http:Payload` annotation on the return type can be used to overwrite the `Content-type` header. Return `anydata` type from the resource method when you want to send the desired payload with the default status code and headers.

Reviewer: ```suggestion
Returning an `anydata` type from the resource method results in an HTTP response where the returned value becomes the body. If the returned type is `nil`, then a `202 Accepted` response is returned to the client without the body. Otherwise, the response contains the returned value as the payload, and the `Content-type` header will be inferred from the return type. In addition, the response will be `201 Created` for `POST` resources and `200 Ok` for other resources. Furthermore, the `@http:Payload` annotation on the return type can be used to override the `Content-type` header. Returning `anydata` type from the resource method is useful when the desired payload with the default status code and headers needs to be sent as the response.
```

Author: Fixed via [9c3c207](https://github.com/ballerina-platform/ballerina-distribution/pull/3769/commits/9c3c207d6ca8971110fd8bce55edd27c8b13e18b)
